helix-term/src/keymap.rs

```diff
@@ -68,7 +68,7 @@ impl KeyTrieNode {
     }
 
     pub fn infobox(&self) -> Info {
-        let mut body: Vec<(&str, BTreeSet<KeyEvent>)> = Vec::with_capacity(self.len());
+        let mut body: Vec<(&str, Vec<KeyEvent>)> = Vec::with_capacity(self.len());
```

Comment:
This seems to revert https://github.com/helix-editor/helix/pull/952

Reply:
Posted images disagree.